src/vconsole/vconsole-setup.c

```diff
@@ -225,7 +225,7 @@ static void setup_remaining_vcs(int fd, bool utf8) {
                 return;
         }
 
-        fontbuf = malloc(cfo.width * cfo.height * cfo.charcount / 8);
+        fontbuf = malloc(32 / 8 * 32 * 512);
```

Comment:
i think a comment explaining where these pretty much random numbers come from would be appropriate here...